https://src.fedoraproject.org/rpms/lyx/blob/rawhide/f/0002-Fix-compilation-on-gcc-15.patch
From: =?UTF-8?q?Jos=C3=A9=20Matos?= <jamatos@lyx.org>
Date: Fri, 17 Jan 2025 07:39:51 +0000
Subject: [PATCH] Fix compilation on gcc-15

Suggested by gcc itself. :-)

(cherry picked from commit 527d3d669cf60d306f6c6a08a880fa4609df2262)
--- a/src/mathed/MathParser.cpp
+++ b/src/mathed/MathParser.cpp
@@ -78,6 +78,7 @@ following hack as starting point to write some macros:
 #include "support/debug.h"
 #include "support/docstream.h"
 
+#include <cstdint>
 #include <sstream>
 
 //#define FILEDEBUG
-- 
2.48.1

